frontend/src/components/DomainDetails.tsx

```diff
@@ -97,31 +97,50 @@ export const DomainDetails: React.FC<Props> = (props) => {
     );
   }, [domain]);
 
+  function days(daytype: any) {
+    return differenceInCalendarDays(
+      Date.now(),
+      parseISO(daytype));
+  }
+
   const overviewInfo = useMemo(() => {
     if (!domain) {
       return [];
     }
     const ret = [];
+
+  let firstSeenDays = days(domain.createdAt);
+
     if (domain.ip) {
       ret.push({
         label: 'IP',
         value: domain.ip
       });
     }
-    ret.push({
-      label: 'First Seen',
-      value: `${differenceInCalendarDays(
-        Date.now(),
-        parseISO(domain.createdAt)
-      )} days ago`
-    });
-    ret.push({
-      label: 'Last Seen',
-      value: `${differenceInCalendarDays(
-        Date.now(),
-        parseISO(domain.updatedAt)
-      )} days ago`
-    });
+
+    (firstSeenDays > 1) ?
+      ret.push({
```

Comment:
This is duplicating a lot of code. Can you instead do:

```
    formatDistanceToNow(date, {addSuffix: true})
```

This will format the given date automatically as "5 days ago"

See https://date-fns.org/v2.20.1/docs/formatDistanceToNow